source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,286 @@
+
+Connect to your instance and make your package executable again as we done previously already:
+
+.. code-block:: console
+
+   $chmod 777 concordium-client
+
+Let's check if everything is connected correctly by displaying a random block.
+
+.. code-block:: console
+
+   $./concordium-client block show c1ffec01a42084f4a406d6408ce435621bc5730288a447a4c8325b560e959108 --grpc-port 10001
+
+You should see some block data output.
+
+.. image:: ./images/pb_tutorial_18.png
+   :width: 100 %
+
+.. Note::
+   Port 10001 is open by default on your testnet node to interact with it.
+
+2. Option (advanced users)
+
+This option explains how you can use the concordium client tool locally at your machine and connect remotely to your node running on the server.
+
+**Advantage**: You don't have to transfer files between your local machine and your instance later in the tutorial when we deploy the smart contracts.
+
+**Disadvantage**: You have to adjust the inbound connection rules to your instance.
+
+.. Note::
+   Port 10001 is open by default on your testnet node to interact with it. But you have to open port 10001 in your inbound connection rules for your instance as well. Please open the port only for your IP address. Please be aware of the security implication, it is assumed that you are the only person using that IP address. Please be aware that we recommend this option for advanced users.
+
+.. image:: ./images/pb_tutorial_14.png
+   :width: 100 %
+
+Let's check if everything is connected correctly by displaying a random block.
+
+.. code-block:: console
+
+   $./concordium-client block show c1ffec01a42084f4a406d6408ce435621bc5730288a447a4c8325b560e959108 --grpc-ip <IP Address of Your Instance> --grpc-port 10001
+
+You should see some block data output.
+
+.. image:: ./images/pb_tutorial_17.png
+   :width: 100 %
+
+
+Create a mobile wallet account on testnet
+=========================================
+
+You can create an account with any of the officially provided wallets from Concordium. This tutorial focuses on the mobile wallet but feel free to explore the setup of the :ref:`desktop wallets<overview-account-desktop>` alternatively.
+
+You can download the mobile wallet package from the :ref:`installation page<downloads-mobile-wallet-testnet>`.
+If you follow the instruction in the mobile app you are asked to name your account and to name your initial identity card. This tutorial uses `Concordium` for the account name and `Account1` for the identity card name but feel free to chose your own naming.
+
+
+.. image:: ./images/pb_tutorial_1.png
+   :width: 20 %
+
+.. image:: ./images/pb_tutorial_2.png
+   :width: 20 %
+
+.. image:: ./images/pb_tutorial_3.png
+   :width: 20 %
+
+
+You don't have to provide an ID to create an account on testnet when selecting `Concordium testnet IP`. This allows you to insert some dummy identity data and is meant for testnet testing only.
```

Comment:
The default dummy data is valid, so you don't even need to change it 😊

Reply:
I changed to
"This gives you an example for a dummy identity that you can customise or use as it is. Dummy identities are meant for testnet testing only."